docs/en/setup/backend/backend-graalvm.md

```diff
@@ -0,0 +1,45 @@
+# How to use SkyWalking with GraalVM native-image (Experimental)
+
+## Native-Image
+Native Image is a technology to ahead-of-time compile Java code to a standalone executable, called a native image. 
+This executable includes the application classes, classes from its dependencies, runtime library classes, 
+and statically linked native code from JDK. It does not run on the Java VM, but includes necessary components like memory management, thread scheduling, and so on from a different runtime system, called “Substrate VM”. 
+Substrate VM is the name for the runtime components (like the deoptimizer, garbage collector, thread scheduling etc.).
+The resulting program has faster startup time and lower runtime memory overhead compared to a JVM.
+
+SkyWalking currently offers support for native-image. However, please note that the OAP started in this manner does not have the same functionality as the regular OAP, and some features are not yet supported.
+
+## Compile Guide
+Notice: If you are not familiar with the compilation process, please read [How-to-build](https://skywalking.apache.org/docs/main/next/en/guides/how-to-build/) first.
+
+The native-image compilation is not enabled by default. To enable it, we need to specify `-p native` during compilation, such as:
+
+```shell
+./mvnw -P native clean package -Dmaven.test.skip
+```
+
+Then, All packages are in `distribution/graal/dist` (.tar.gz for Linux and .zip for Windows).
```

Comment:
I think we should be clear about pre-native and native tarballs
And, I don't see anything about .zip and windows. Please be careful about what you are writing.